content/standards/index.html.md.erb

```diff
@@ -33,5 +33,8 @@ The Data Standards Catalogue currently contains the following Standards:
   </table>
 </div>
 
+## External Standards Catalogues
+You can find a list of external standards catalogues [here](/standards/external-standards.html).
```

Comment:
:question: I've read before (sorry I can't remember where!) that `here` isn't the most accessible for screen readers - could we reword the sentence so the link can make more sense?

Reply:
is this ok? https://github.com/alphagov/data-standards-authority/pull/100/commits/bdaabd3c0a6bcae072b1e48e47ab7297338f13ab